app/__init__.py

```diff
@@ -1,11 +1,43 @@
 import os
 from flask import Flask, render_template, request
 from dotenv import load_dotenv
+import folium
+import json
 
 load_dotenv()
 app = Flask(__name__)
 
 
 @app.route('/')
 def index():
-    return render_template('index.html', title="MLH Fellow", url=os.getenv("URL"))
+    # Create a map object
+    m = folium.Map(min_zoom=2)
```

Comment:
I think using a more descriptive variable name than "m" might be a little cleaner 🧹

Reply:
I'll get right into it!